src/tabs/calculations.jsx

```diff
@@ -100,6 +102,10 @@ function Calculations() {
                 afterDamageProps: Context.afterDamageProps
             }
 
+            if (!blocked && res.block || !missed && res.miss) {
+                continue;
```

Comment:
It may be a better idea to disabling missing and blocking entirely in the damage calculation, as this will just not include missed or blocked hits in the results. If you have 180 missed hits out of the simulated 200 hits, you will only be able to see 20 results.
https://github.com/Frostiae/Flyffulator/blob/229d476c0b5fbc67e67bcaae1c56e70ddbc88018/src/flyff/flyffdamagecalculator.js#L60